src/main/java/org/jitsi/videobridge/rest/ColibriWebSocketServlet.java

```diff
@@ -26,6 +27,7 @@
 /**
  * @author Boris Grozev
  */
+@SuppressFBWarnings("SE_BAD_FIELD")
```

Comment:
Reading the description of this warning, I wonder if it's wonder we should disable overall?